src/js/node/fs.ts

```diff
@@ -131,7 +135,11 @@ class StatWatcher extends EventEmitter {
   start() {}
 
   stop() {
-    this._handle?.close();
+    if (!this._handle) return;
+
+    process.nextTick(emitEventNT.bind(this, "stop"));
```

Comment:
usually instead of calling .bind we make a specific function and do something like
`process.nextTick(emitStopNT, this);`
eg:
https://github.com/nodejs/node/blob/v24.3.0/lib/internal/fs/watchers.js#L154
https://github.com/nodejs/node/blob/v24.3.0/lib/internal/fs/watchers.js#L74